docs/index.rst

```diff
@@ -9,7 +9,7 @@ you write cross platform native GUI Python applications. It includes:
 
 * `Toga <https://toga.readthedocs.io>`__, a cross platform widget toolkit;
 * `Briefcase <https://briefcase.readthedocs.io>`__, a tool for packaging Python
-  projects as distributable artefacts that can be shipped to end users;
+  projects as distributable artifacts that can be shipped to end users;
```

Comment:
This is a hill I will die on :-) Artefact comes from the Latin arte factum. It's the commonwealth-preferred spelling, and dammit, it's *correct*.

Reply:
:) That's definitely fine with me; I think I was over-interpreting this [comment](https://github.com/beeware/toga/pull/1854#pullrequestreview-1377038053):

> As for the UK/AU spelling - for the sake of consistency and avoiding the inevitable "you spelled colour wrong" pull requests, we should converge on US spelling.

Reply:
Oh - in almost every other case, I'll sigh and begrudgingly accept the US spelling. But not for artefact. Y'all are just wrong on that one. :-) 

It might be worth adding artefact specifically as the one exception to avoid future confusion...